mod-orders/schemas/bindPiecesResult.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Result of Binding Pieces",
+  "type": "object",
+  "properties": {
+    "poLineId": {
```

Comment:
what about adding status. whether it is success or fail

Reply:
using some of the fields of receiving Results if possible https://github.com/folio-org/acq-models/blob/866db73c9202864e45eb767f7ee765986f3f3c0f/mod-orders/schemas/receivingResults.json#L38

Reply:
after discussion, success field can be redundant